app/controllers/application_controller.rb

```diff
@@ -9,4 +9,5 @@ def addE(action)
     end
     action
   end
+
```

Comment:
Can we get rid of this? There's no point in having a file whose only change was a single new line. It doesn't look like it improves readability at all.